yarn.lock

```diff
@@ -14,6 +14,13 @@
   dependencies:
     ses "^0.12.7"
 
+"@agoric/assert@^0.3.11":
```

Comment:
@kriskowal is this like Agoric/dapp-fungible-faucet#31 (comment) , a RESM to NESM thing?

Reply:
@katelynsills Yup. Came back after I reverted it and did "agoric install".

Reply:
Not likely to be related.

Reply:
Though, if `agoric install` causes an update to `yarn.lock`, I do recommend including a commit with just those changes. It would be cleaner if `yarn.lock` is always up-to-date on HEAD, and for that to be an orthogonal PR, but I’ve just been healing the lockfile *before* stacking my changes, then healing the lockfile again after my changes to separate those that are deliberate.

At $job-1, we had a CI job that ensured that `yarn.lock` was untouched after `yarn install` (or rather, its spiritual predecessor).

Reply:
Oh, it all makes sense anyway. I added a new dependency on `@agoric/marshal`.

Reply:
> @katelynsills Yup. Came back after I reverted it and did "agoric install".

Ok, it looks like these are the result of including `marshall`. Looks good to me!